MANIFEST.in

```diff
@@ -5,5 +5,6 @@ recursive-include   doc         *
 recursive-include   examples    *
 recursive-include   rpm         *
 recursive-exclude   leakcheck   *.py *.pem
+recursive-exclude   examples    *.crt *.key *.cert *.pkey
```

Comment:
yep! ;)  but it prevents the tests to fail under certain circumstances (some I am in right now).  I can remove it tho to not spoil this special moment. ;)